src/js/components/covid19/OpportunityProjectPage.jsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * OpportunityProjectPage.jsx
+ * Created by Brett Varney 10/27/21
+ */
+
+import React from 'react';
+import PageWrapper from 'components/sharedComponents/PageWrapper';
+import { topPageMetaTags } from 'helpers/metaTagHelper';
+import { FlexGridContainer, FlexGridRow, FlexGridCol } from 'data-transparency-ui';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useDispatch } from 'react-redux';
+import { showModal } from 'redux/actions/modal/modalActions';
+
+require('pages/covid19/opportunityProject.scss');
+
+const OpportunityProjectPage = () => {
+    const dispatch = useDispatch();
+    const handleClick = (e) => {
+        e.preventDefault();
+        dispatch(showModal(e.target.href));
+    };
+
+    const externalLink = (href, linkText) => (
+        <a href={href} onClick={handleClick}>
+            {linkText}&nbsp;
+            <FontAwesomeIcon size="sm" icon="external-link-alt" />
+        </a>
+    );
+
+    return (
+        <PageWrapper
+            pageName="The Opportunity Project"
+            title="The Opportunity Project"
+            noHeader
+            metaTagProps={topPageMetaTags}>
+            <FlexGridContainer className="top-page">
+                <FlexGridRow>
+                    <FlexGridCol desktop={{ span: 8, offset: 2 }} className="content">
+                        <main role="main" className="main-content">
+                            <header className="main-heading">Analyzing Equity in Federal COVID-19 Spending: The Opportunity Project
```

Comment:
It's recommended to have only one h1 on a page - normally on our site this is in the gray bar that was removed from this design. But on this page I think it makes sense for this to be the h1, and the headings below to be h2s.
https://dequeuniversity.com/rules/axe/4.1/page-has-heading-one
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header
```suggestion
                            <header className="main-heading">
                                <h1>Analyzing Equity in Federal COVID-19 Spending: The Opportunity Project</h1>
```

Reply:
done